src/NLog/Layouts/Log4JXmlEventLayout.cs

```diff
@@ -58,6 +58,26 @@ public Log4JXmlEventLayout()
         public Log4JXmlEventLayoutRenderer Renderer { get; private set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to include contents of the <see cref="MappedDiagnosticsContext"/> dictionary.
+        /// </summary>
+        /// <docgen category='Payload Options' order='10' />
+        public bool IncludeMdc { get { return Renderer.IncludeMdc; } set { Renderer.IncludeMdc = value; } }
```

Comment:
maybe we should add an interface that dictates the properties shared by Log4JXmlEventLayout.cs and Log4JXmlEventLayoutRenderer.cs ? 

Then it's more clear those are coupled. What do you think?